gremconnect/response.go

```diff
@@ -53,9 +53,8 @@ func MarshalResponse(msg []byte) (Response, error) {
 		code   = status["code"].(float64)
 		resp   = Response{Code: int(code)}
 	)
-	message, _ := status["message"].(string)
 
-	err = responseDetectError(resp.Code, message)
```

Comment:
Changes in this file were removed [here](https://gitlab.com/kaleidoscope.root/grammes-gremlin/-/commit/58aecaa0988a60f068717bef400c984533914001)

should be put back imo